integration/kubernetes/helm-chart/alluxio/Chart.yaml

```diff
@@ -12,7 +12,7 @@
 name: alluxio
 apiVersion: v1
 description: Open source data orchestration for analytics and machine learning in any cloud.
-version: 0.6.20
+version: 0.6.21
```

Comment:
Please rebase on the latest `master` as the version has gone up after a few changes, thanks!